recipes/nbpresent/run_test.sh

```diff
@@ -0,0 +1,11 @@
+
+# On circle-ci, we are building in a container, so CIRCLE_CI is not set
+# So this will have to do.
+if [ `which yum` ]; then
+  yum install -y fontconfig freetype
```

Comment:
This seems really brittle... any suggestions for doing this in a better place? Is there a way for a recipe to request more packages?